src/modules/gimbal/output.h

```diff
@@ -89,6 +89,7 @@ class OutputBase
 	void _calculate_angle_output(const hrt_abstime &t);
 
 	float _angle_outputs[3] = { 0.f, 0.f, 0.f }; ///< calculated output angles (roll, pitch, yaw) [rad]
```

Comment:
Could you rather update `_angle_outputs` to include the compensation?

Reply:
I have not checked completely, but from what I see it's using the same logic with the params.
How would it break the feedback?
Isn't the mavlink output broken in the same way if you were to enable `MNT_DO_STAB`?
If we have good reasons to do it differently, I don't mind, but it needs to be clear why.